src/services/Task/BoundedTask.js

```diff
@@ -43,7 +43,108 @@ export const receiveBoundedTasks = function(tasks,
   }
 }
 
-// async action creators
+/**
+ * Retrieve all task markers (up to the given limit) matching the given search
+ * criteria, which should at least include a boundingBox field, and may
+ * optionally include a filters field with additional constraints
+ */
+export const fetchBoundedTaskMarkers = function(criteria, limit = 50, skipDispatch = false, ignoreLocked = true) {
```

Comment:
Nit: you can write `export function fetchBoundedTaskMarkers(criteria, ...) {` instead; it's shorter.